bianca_pipes.rb

```diff
@@ -0,0 +1,202 @@
+##### Wave_1 #####
```

Comment:
For a project with waves like this, each wave should build on the last - you don't need to save the intermediate results.